### PR TITLE
feat(doc): new logo and screenshots

### PR DESCRIPTION
**Pull request by alextran1502**

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -88,11 +88,10 @@ const config = {
         },
       },
       navbar: {
-        title: 'IMMICH',
```
Comment (jrasm91) [inline docs/docusaurus.config.js:91]: Why did you remove the title? Did you mean to use an immich with the text in it? Right now it no longer appears and I think it should be added back.

Comment (alextran1502) [reply docs/docusaurus.config.js:91]: Yeah, I can add the logo with text. The logo with text is vectorized, we don't have the license to use Router Medium directly as a font on the web, but its vectorized form and images is fine
